tests/test-dt-stats.py

```diff
@@ -504,7 +504,7 @@ def test_issue1953():
 
 def test_stats_bool_large(numpy):
     n = 12345678
-    a = numpy.random.randint(2, size=n, dtype=numpy.bool8)
+    a = numpy.random.randint(2, size=n, dtype=numpy.bool_)
```

Comment:
Isn't it [the same thing](https://numpy.org/doc/stable/reference/arrays.scalars.html#numpy.bool8)?

Reply:
numpy has deprecated the numpy.bool8 function

Reply:
[deprecation link](https://numpy.org/devdocs/release/1.24.0-notes.html#np-str0-and-similar-are-now-deprecated)

Reply:
I see, thanks.